app/lib/package/api_export/exported_api.dart

```diff
@@ -270,6 +270,67 @@ final class ExportedPackage {
         Duration(hours: 2),
       );
 
+  Future<void> synchronizeTarballs(
```

Comment:
Let's document what the input parameter and with this method does, including the GC part.